code/ZAS/Fire.dm

```diff
@@ -269,11 +269,13 @@ Attach to transfer valve and open. BOOM.
 	dir = pick(cardinal)
 	set_light(3)
 	SSair.add_hotspot(src)
+	score["turfsonfire"]++
```

Comment:
Just above this, there's a line that adds the fire to `SSair.processing_parts[SSAIR_HOTSPOT]`. You can just read the length of that list instead of manually keeping count.

Reply:
Done